src/views/Player.vue

```diff
@@ -226,9 +237,22 @@ export default {
           this.isFullscreen = false;
           this.sessionId = res.data.sessionId;
           this.browser = res.data.userAgent["browser"]["family"];
-          this.configs = res.data.configData;
-          this.isTutorialComplete = this.configs.tutorial.isComplete;
-          this.tutorialProgress = this.configs.tutorial.progress;
+          this.userConfigs = res.data.userConfig;
+          this.isTutorialComplete = this.userConfigs.tutorial.isComplete;
+          this.tutorialProgress = this.userConfigs.tutorial.progress;
```

Comment:
Changed `configData` to `userConfig` because now we also have `plioConfig` so `plioConfig` would've been hard to differentiate from `configData`

Reply:
👍🏾